lib/tessel/access-point.js

```diff
@@ -46,14 +47,46 @@ Tessel.prototype.enableAccessPoint = function() {
 Tessel.prototype.disableAccessPoint = function() {
   var status = 'Access Point successfully disabled.';
 
-  return new Promise((resolve) => {
+  return new Promise((resolve, reject) => {
     return this.simpleExec(commands.turnAccessPointOff())
       .then(() => {
-        return commitAndClose(this, status, resolve);
+        return commitAndClose(this, status, resolve, reject);
       });
   });
 };
 
+Tessel.prototype.getAccessPointInfo = function() {
+  var info;
+
+  return new Promise((resolve, reject) => {
+    return this.simpleExec(commands.getAccessPointConfig())
+      .then((config) => {
+        const props = ['ssid', 'key', 'encryption', 'disabled'];
+        const values = props.map((prop) => {
+          const regex = new RegExp(`${prop}='(.+)'`);
+          const value = regex.exec(config);
+
+          return value ? value[1] : null;
+        });
+
+        info = {
+          ssid: values[0],
+          password: values[1],
+          security: values[2],
+          disabled: values[3]
+        };
```

Comment:
> What happens when config is not what we expected?

As line 69 shows, if the regex doesn't find the prop it expects from `config`, then `value` would be undefined and set to `null` for that prop. In `lib/controller.js` I check to see if `.ssid` is set before logging the info returned by `getAccessPointInfo`. Reference -> https://github.com/tessel/t2-cli/pull/596/files#diff-f68567477d803b49930337bf7fe1556bR749